### PR TITLE
Add perf pipeline pr triggers

### PR DESCRIPTION
**Pull request by adiaaida**

### CODE REVIEW

**Comment by adiaaida** [comment]
@dotnet-bot test ci please

**Comment by adiaaida** [comment]
@dotnet-bot test ci please

**Comment by adiaaida** [comment]
@dotnet-bot test ci please

**Comment by adiaaida** [comment]
@dotnet-bot test ci please

**Comment by adiaaida** [comment]
@dotnet/rap-team PTAL

File: buildpipeline/perf-pipeline.groovy [hunk 1]
```diff
@@ -7,188 +7,200 @@
...
-        String baselineString = ""
-        if (isBaseline) {
-            baselineString = "-baseline"
-        }
-        dir ('.') {
-            unstash "nt-${arch}-${pgo}${baselineString}-build-artifacts"
-            unstash "benchview-tools"
-            unstash "metadata"
-        }
+    simpleNode('windows_server_2016_clr_perf', 180) {
```
Comment (adiaaida) [inline buildpipeline/perf-pipeline.groovy]: Note: This would mean ILLink jobs would go on the perf machines. Not sure if we actually want that or not.

Comment (jorive) [reply buildpipeline/perf-pipeline.groovy]: I am updating the label on https://github.com/dotnet/coreclr/pull/14900. This was illink will run on VMs.

Comment (adiaaida) [reply buildpipeline/perf-pipeline.groovy]: Right, but I've moved the simpleNode lines out of the individual calls to windowsPerf, etc. into windowsPerf, so if we want a different node type for illink I'll have to do some experimenting/changing of how we do this part.

Comment (jorive) [reply buildpipeline/perf-pipeline.groovy]: We definitely do not want to use real hardware where it is not needed.

Comment (adiaaida) [reply buildpipeline/perf-pipeline.groovy]: Ok, I'll try to finagle something so that we can use something different for illink.

**Comment by RussKeldorph** [comment]
@adiaaida Is this PR still active?  Could you set the milestone if so?

**Comment by adiaaida** [comment]
@RussKeldorph no, sorry. This got merged into a different PR. I forgot to close this version.

**Pull request closed without merging**